utils/zerovec/derive/src/make_varule.rs

```diff
@@ -600,6 +626,9 @@ impl<'a> UnsizedFieldKind<'a> {
         match *ty {
             Type::Reference(ref tyref) => OwnULETy::new(&tyref.elem, "reference").map(UnsizedFieldKind::Ref),
             Type::Path(ref typath) => {
+                if let Some(custom_varule_ident) = custom_varule_ident {
+                    return Ok(UnsizedFieldKind::Custom(typath, custom_varule_ident));
```

Comment:
Question: is it correct that previously, this only supported the exact matches ZeroVec, VarZeroVec, Cow, Vec, and Box, but now it supports things annotated with `zerovec::varule` as well?

Reply:
correct